symfony/client/src/orangehrmPimPlugin/pages/employee/EmployeeTaxExemption.vue

```diff
@@ -0,0 +1,229 @@
+<!--
+/**
+ * OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+ -->
+
+<template>
+  <edit-employee-layout :employee-id="empNumber" screen="tax">
+    <div class="orangehrm-horizontal-padding orangehrm-vertical-padding">
+      <oxd-text tag="h6" class="orangehrm-main-title">
+        Tax Exemptions
+      </oxd-text>
+      <oxd-divider />
+      <oxd-form :loading="isLoading" @submitValid="onSave">
+        <oxd-text class="orangehrm-sub-title" tag="h6">
+          Federal Income Tax
+        </oxd-text>
+        <oxd-divider />
+        <oxd-form-row>
+          <oxd-grid :cols="3" class="orangehrm-full-width-grid">
+            <oxd-grid-item>
+              <oxd-input-field
```

Comment:
Add FE Permissions to all input fields. Refer [PIM - Job](https://github.com/orangehrm/orangehrm/blob/5.0/symfony/client/src/orangehrmPimPlugin/pages/employee/EmployeeJob.vue#L38)